### PR TITLE
[xml] Update portuguese.xml

### PR DESCRIPTION
**Pull request by hugok79**
@donho @horus68 https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15682

### CODE REVIEW

File: PowerEditor/installer/nativeLang/portuguese.xml [hunk 1]
```diff
@@ -630,14 +630,16 @@ Nota de tradução:
...
+					<Item id="2226" name="Forçar cor de primeiro plano para todos os estilos"/>
+					<Item id="2227" name="Forçar cor de plano de fundo para todos os estilos"/>
+					<Item id="2228" name="Forçar tipo de letra para todos os estilos"/>
+					<Item id="2229" name="Forçar tamanho de tipo de letra para todos os estilos"/>
+					<Item id="2230" name="Forçar tipo de letra em negrito para todos os estilos"/>
+					<Item id="2231" name="Forçar tipo de letra em itálico para todos os estilos"/>
+					<Item id="2232" name="Forçar tipo de letra sublinhada para todos os estilos"/>
+					<Item id="2234" name="Ir para definições"/>
+					<!-- Don't translate "&quot;Global override&quot; -->
+					<Item id="2235" name="O que é Substituição global?"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/portuguese.xml:642]: "Global override" should not be translated.

**Review by donho — CHANGES_REQUESTED** [review]
"Global override" and "Default Style" should not be translated.

File: PowerEditor/installer/nativeLang/portuguese.xml [hunk 2]
```diff
@@ -1813,6 +1821,8 @@ Clique no botão &quot;?&quot; à direita para abrir o site com o Manual do Util
 C, C++, Java, C#, Objective-C, PHP, JavaScript, JSP, CSS, Perl, Rust, PowerShell e JSON.
 
 Se selecionar o modo avançado mas não editar ficheiros nas linguagens acima mencionadas, a indentação manter-se-á no modo básico."/>
+			<!-- Don't translate "&quot;Global override&quot; and &quot;Default Style&quot; -->
+			<global-override-tip value="Ativar a &quot;Substituição global&quot; aqui irá substituir esse parâmetro em todos os estilos de linguagem. O que provavelmente pretende é usar as definições de &quot;Estilo predefinido&quot; em vez disso"/>
```
[the remark this answers is not in the document]
Comment (hugok79) [reply PowerEditor/installer/nativeLang/portuguese.xml]: Done. Thank you!

File: PowerEditor/installer/nativeLang/portuguese.xml (hunk 1 above)
Comment (hugok79) [reply PowerEditor/installer/nativeLang/portuguese.xml]: Done. Thank you!

**Review by donho — APPROVED** [review]

**Pull request closed without merging**